addon/pods/components/frost-tab/template.hbs

```diff
@@ -1 +1,15 @@
-{{yield}}
+<div class='tab' data-test={{hook (concat hook '-tab' (if isSelected '-selected' '')) }}>
+  {{frost-button
+    class=(concat 'tab' (if isSelected ' active'))
```

Comment:
dont need the concat right? `class= 'tab (if isSelected "active")'`? Or use a computed.